### ISSUE TITLE
Checkbox is not set as touched

### ISSUE THREAD

**Issue by iamvanja** [jannikbuschke/formik-antd#56]
Consider 
```js
<Formik
  ...
  render={() => (
    <Form>
      <FormItem name='rememberMe'>
        <Checkbox name='rememberMe'>
          Remember me
        </Checkbox>
      </FormItem>
    </Form>
  )}
/>
```

Toggling this checkbox does not set `rememberMe` field as touched.

**Comment by jannikbuschke** [comment]
fixed in v1.1

**Comment by iamvanja** [comment]
Appreciate the quick resolution on this, but I don't think this is fully resolved.
To allow custom onChange while still internally updating Formik state, prop onChange should not overwrite existing onChange. Instead, existing (internal) onChange should check if the user onChange is passed as a prop and execute it if so.

So instead of 
```js
...
onChange={v => {
  setFieldValue(name, v.target.checked)
  setFieldTouched(name, true)
}}
{...restProps}
...
```

something like this

```js
...
{...restProps}
onChange={v => {
  setFieldValue(name, v.target.checked)
  setFieldTouched(name, true)
  restProps.onChange && restProps.onChange(v)
}}
...
```
